src/systemctl/systemctl.c

```diff
@@ -5666,6 +5706,12 @@ static int enable_unit(int argc, char *argv[], void *userdata) {
                 const char *method;
                 sd_bus *bus;
 
+                r = unit_exists(*names);
+                if (r < 0)
+                        return r;
+                if (r == 0)
+                        log_notice("Unit %s does not exist, proceeding anyway.", *names);
+
```

Comment:
where did the check for the "mask/unmask" verb go? the chunk above should only be done for "mask" and "unmask"!